lxc/image.go

```diff
@@ -863,7 +863,11 @@ func (c *cmdImageImport) run(cmd *cobra.Command, args []string) error {
 	opAPI := op.Get()
 
 	// Get the fingerprint
-	fingerprint := opAPI.Metadata["fingerprint"].(string)
+	fingerprint, ok := opAPI.Metadata["fingerprint"].(string)
+	if !ok {
+		return fmt.Errorf("Invalid type %T for \"fingerprint\" key in operation metadata", fingerprint)
```

Comment:
Please can you use backticks when wanting to use " in strings to avoid having to escape them, so:

```go
return fmt.Errorf(`Invalid type %T for "fingerprint" key in operation metadata`, fingerprint)
```